### PR TITLE
Add `roave/security-advisories` as suggested dependency

### PR DESCRIPTION
**Pull request by Ocramius**
It is a very simple addition that leads to the exclusion of vulnerable dependencies
directly when `composer update` is running

### CODE REVIEW

**Comment by paragonie-scott** [comment]
This was one of my suggestions in #3 :)

**Comment by Ocramius** [comment]
@paragonie-scott sorry, didn't see it before ^_^

File: README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ Supported by: [GuardRails.io](https://github.com/apps/guardrails)
 - [security-checker](https://github.com/sensiolabs/security-checker) - PHP frontend for security.symfony.com.
   - `docker pull guardrails/security-checker`
 - [Symfony Security Monitoring](https://security.symfony.com/) - PHP security vulnerabilities monitoring.
+- [`roave/security-advisories`](https://github.com/Roave/SecurityAdvisories) - Add this dependency to disallow known/vulnerable installation of packages directly through `composer update`
```
Comment (streichsbaer) [inline README.md]: Please remove the backticks in the link name, so it's the same format as the others.

**Review by streichsbaer — CHANGES_REQUESTED** [review]
Thanks for the contribution @Ocramius. 
Please just remove the backticks as per the comment below, then it's good to be merged.

File: README.md (hunk 1 above)
Comment (Ocramius) [reply README.md]: @streichsbaer done :+1:

**Review by streichsbaer — APPROVED** [review]

**Pull request merged**